acct-group/i2p/i2p-0.ebuild

```diff
@@ -0,0 +1,9 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+DESCRIPTION="Group for system-wide net-vpn/i2p"
+ACCT_GROUP_ID=493
```

Comment:
You had a reply in gentoo-dev about switching this, please either take 471 or the others that were proposed.